.github/workflows/main.yml

```diff
@@ -10,8 +10,11 @@ defaults:
   run:
     shell: bash
 
-jobs:
+env:
+  DOCKERHUB_REPO: postgis/postgis
+  GITHUB_REPO: postgis/docker-postgis
```

Comment:
(optional) add  env :   `LATEST_VERSION: 17-3.5`    
-  to define which build is tagged as latest, and implement the related tagging.